app/controllers/routeDetails/routing/AddAnotherCountryOfRoutingController.scala

```diff
@@ -50,7 +50,7 @@ class AddAnotherCountryOfRoutingController @Inject() (
     implicit request =>
       val (countries, numberOfCountries, allowMoreCountries) = viewData
       numberOfCountries match {
-        case 0 => Redirect(routes.BindingItineraryController.onPageLoad(lrn, NormalMode))
+        case 0 => Redirect(routes.AddCountryOfRoutingYesNoController.onPageLoad(lrn, NormalMode))
```

Comment:
I believe this change is incorrect. There is a scenario in which the user can start adding countries of routing without ever having to answer the "add country of routing yes/no" question. Redirecting them to here would go against our navigation logic. Sending them to binding itinerary ensures that whatever they then subsequently answer for that question will take them to the correct place